docs/data/sdks/marketing-analytics-browser/index.md

```diff
@@ -13,9 +13,9 @@ The Marketing Analytics Browser SDK extends the Browser SDK to identify users an
 
     [:material-github: GitHub](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/marketing-analytics-browser) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-TypeScript/releases?q=marketing-analytics-browser&expanded=true) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-TypeScript/modules/_amplitude_marketing_analytics_browser.html)
 
-!!!note "Marketing Analytics Browser versus the Typescript Broswer SDK"
+!!!note "Marketing Analytics Browser versus the Broswer SDK"
```

Comment:
Suggestion + typo
```suggestion
!!!note "Marketing Analytics SDK versus the Browser SDK"
```

Reply:
Probably, it should be  `Marketing Analytics Browser SDK`?